cmd/apiserver-boot/boot/create_resource.go

```diff
@@ -183,6 +186,8 @@ import (
 )
 
 // +genclient=true
+{{- if .NonNamespacedKind }}
+// +nonNamespaced=true{{ end }}
```

Comment:
Instead of setting this and changing the code generator, why not just override the strategy to have the namespace == false?